homeassistant/components/dsmr/sensor.py

```diff
@@ -426,7 +426,7 @@ class MbusDeviceType(IntEnum):
     DSMRSensorEntityDescription(
         key="actual_threshold_electricity",
         translation_key="actual_threshold_electricity",
-        obis_reference="ACTUAL_TRESHOLD_ELECTRICITY",
+        obis_reference="ACTUAL_TRESHOLD_ELECTRICITY", # Mispelt in external tool
```

Comment:
```suggestion
        obis_reference="ACTUAL_TRESHOLD_ELECTRICITY",
```

Mispelt was Misspelled. And we avoid sich comments.